src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,51 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string,
+  description: string,
+  imgUrl: string,
+  imdbUrl: string,
+};
```

Comment:
```suggestion
type Props = {
  movie: Movie;
};
```
Use your `Movie` interface here